speaker/bgp_controller.go

```diff
@@ -60,6 +59,15 @@ type peer struct {
 	session bgp.Session
 }
 
+func (p peer) id() string {
```

Comment:
same as above, can we set this when we create an instance of `peer` ?

Reply:
done